src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -115,13 +115,17 @@ def load_arguments(self, _):
         c.argument('os_type', arg_type=get_enum_type(OperatingSystemTypes), help='The Operating System type of the Disk.')
         c.argument('disk_iops_read_write', type=int, min_api='2018-06-01', help='The number of IOPS allowed for this disk. Only settable for UltraSSD disks. One operation can transfer between 4k and 256k bytes')
         c.argument('disk_mbps_read_write', type=int, min_api='2018-06-01', help="The bandwidth allowed for this disk. Only settable for UltraSSD disks. MBps means millions of bytes per second with ISO notation of powers of 10")
+        c.argument('upload_size_byte', type=int, min_api='2019-03-01',
```

Comment:
i think better name should be "upload_size_bytes"

Reply:
OK